antenna-documentation/Jenkinsfile.eclipse.documentation

```diff
@@ -25,34 +25,17 @@ pipeline {
     environment {
         GITHUB_CREDENTIALS = credentials('7d588c11-a40f-4529-bcaa-57b6d9691af6')
     }
-
     stages {
-        stage('Checkout') {
-            steps {
-                git branch: 'neubs-bsi/doc/hostWithGhPages', credentialsId: '7d588c11-a40f-4529-bcaa-57b6d9691af6', url: 'https://github.com/bsinno/antenna.git'
-            }
-        }
         stage('Configure Git') {
             steps {
                 sh 'git config --global user.name "${GITHUB_CREDENTIALS_USR}"'
                 sh 'git config --global user.author "${GITHUB_CREDENTIALS_USR}"'
                 sh 'git config --global user.email "${GITHUB_CREDENTIALS_USR}@eclipse.org"'
             }
         }
-        stage('Build Antenna') {
-            steps {
-                sh 'mvn -B clean install -DskipTests'
-            }
-        }
         stage('Site Deploy') {
             steps {
-                sh '''
-                 GITHUB_USERNAME=${GITHUB_CREDENTIALS_USR} \
-                    GITHUB_PASSWORD=${GITHUB_CREDENTIALS_PSW} \
-                    mvn clean \
-                    -f ./antenna-documentation/pom.xml \
-                    site-deploy -Psite-deploy
-                '''
+                sh 'bash antenna-documentation/scripts/main.sh'
```

Comment:
You should drop the `bash` here to rely on the script's shebang line.

Reply:
need this `bash` to avoid permission denied of `main.sh`

Reply:
That shouldn't be necessary if you commit `main.sh` with `x`set already in Git. Also see https://git-scm.com/docs/git-update-index#Documentation/git-update-index.txt---chmod-x.